client/extensions/woocommerce/state/ui/payments/methods/test/selectors.js

```diff
@@ -0,0 +1,415 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getPaymentMethodEdits,
+	getPaymentMethodsWithEdits,
+	getCurrentlyEditingPaymentMethod,
+	getPaymentMethodsGroup,
+	isCurrentlyEditingPaymentMethod,
+} from '../selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+describe( 'selectors', () => {
+	describe( 'getPaymentMethodEdits', () => {
+		it( 'should return empty array when the methods are being loaded', () => {
+			const state = {
```

Comment:
Maybe you can create a few possible states to test against, rather than adding the states inside each test? See [the settings selector test](https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/state/sites/settings/general/test/selectors.js#L16) — this way tests are easier to scan. (if your state really does need to change between each test, then leaving it as-is makes sense)